src/cli/utils.js

```diff
@@ -49,3 +49,15 @@ exports.getIPFS = (callback) => {
 exports.getRepoPath = () => {
   return process.env.IPFS_PATH || os.homedir() + '/.ipfs'
 }
+
+exports.createLogger = (visible) => {
```

Comment:
Why do you need to `create` the logger? It does not hold any state. You could just have

```js
```